llvm/lib/Analysis/ValueTracking.cpp

```diff
@@ -2816,6 +2816,20 @@ static bool isKnownNonZeroFromOperator(const Operator *I,
       case Intrinsic::bswap:
       case Intrinsic::ctpop:
         return isKnownNonZero(II->getArgOperand(0), DemandedElts, Depth, Q);
+      case Intrinsic::usub_sat: {
+        KnownBits XKnown =
+            computeKnownBits(II->getArgOperand(0), DemandedElts, Depth, Q);
```

Comment:
Please check the compile-time impact of this change.

Reply:
No meaningful regressions: https://llvm-compile-time-tracker.com/compare.php?from=beded9b9ceab19f81320c7cf5e3600a7745c8f05&to=969c66811ee0e1df663503e353b4381e3b7df0b4&stat=instructions:u